### PR TITLE
initial french translation

### PR DESCRIPTION
**Pull request by dlucasd**
Hi,

Here's the french translation of this great template.
Thanks!

### CODE REVIEW

**Comment by gernotstarke** [comment]
Hi there,
thanx for your awesome contribution...

We're briefly reviewing it (I just found a French-speaking colleague)...

It would be good if you could reveal your name, so we can add a "thank you" on the landing page,
when we announce the new version!!
It would be perfectly ok if you prefer not to be named, then it will be an anonymous contribution...

regards from Cologne,
Gernot (co-founder and core maintainer of arc42)

**Comment by gernotstarke** [comment]
thanx @martinweck for providing review comments

**Comment by dlucasd** [comment]
My name is Damien Lucas!

Regards from Bordeaux
Damien

File: FR/asciidoc/src/05_building_block_view.adoc [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+Maintenir une vue d'ensemble de votre code source en rendant sa structure compréhensible grâce à l'abstraction.
+
+Cela vous permet de communiquer avec votre partie prenante à un niveau abstrait sans divulguer les détails de la mise en œuvre.
+
+.Représentation
+La vue en briques est une collection hiérarchique de boîtes noires et de boîtes blanches (voir figure ci-dessous) et de leurs descriptions.
+
+image::05_building_blocks-EN.png["Hiérarchie des briques"]
+
+*Niveau 1* est la description en boîte blanche du système global ainsi que la description en boîte noire de toutes les briques qu'il contient.
```
[the remark this answers is not in the document]
Comment (martinweck) [reply FR/asciidoc/src/05_building_block_view.adoc:26]: I have informed Gernot about it who may adapt it as well

**Review by martinweck — CHANGES_REQUESTED** [review]
Hi Damien,
Thank you so much for your contribution. I am not a native speaker. Hence, if you go through my comments and you feel that they are just wrong, because of my lack of language skills, then just close the comment without action. We have asked also a French-speaking colleague to have a look as well, but for me, it looks all great, only maybe some minor remarks.

File: FR/asciidoc/src/12_glossary.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+* n'utilisent pas de synonymes et d'homonymes
+
+
+.Représentation
+
+Un tableau avec les colonnes <Terme> et <Définition>.
+
+Potentiellement plus de colonnes au cas où vous auriez besoin de traductions.
+
+
```
Comment (martinweck) [inline FR/asciidoc/src/12_glossary.adoc:26]: Maybe, you could add a table here. IMHO it makes it easier for the reader to understand how a glossary could look like, something like:

```
[options="header",cols="1,1"]
|===
|Terme|Définition
| _<Term-1>_ | _<Définition-1>_
| _<Term-2>_ | _<Définition-2>_ | _<Solution-2>_ | _<Lien-2>_
|===
```

**Review by martinweck — APPROVED** [review]
since there are no important things, as with Gernot discussed it's approved

**Review by martinweck — APPROVED** [review]
since there are no important things, as with Gernot discussed it's approved

**Review by martinweck — APPROVED** [review]

**Review by gernotstarke — APPROVED** [review]
Damien, if you like the comments from @martinweck , please feel free to include them.
Otherwise, I gladly accept your PR.

Due to the technical setup of arc42, it might take a few days to have the download available on the official website (we have another person review changes to the downloads... )

Thanx again, merci beaucoup!

File: FR/asciidoc/src/01_introduction_and_goals.adoc [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+* les parties prenantes concernées et leurs attentes
+****
+
+=== Vue d'ensemble des exigences
+
+[role="arc42help"]
+****
+.Contenu
+Brève description des exigences fonctionnelles, des forces motrices, de l'extrait (ou du résumé) des exigences.
+Lien vers (en espérant qu'ils existent) les documents relatifs aux exigences (avec le numéro de version et des informations sur l'endroit où les trouver).
```
[the remark this answers is not in the document]
Comment (dlucasd) [reply FR/asciidoc/src/01_introduction_and_goals.adoc:24]: Sure, "des documents relatifs aux exigences" is more indirect. But "des documents d'exigences" is not quite good.
We debated with my colleagues and "des documents décrivant les exigences" is more accurate

File: FR/asciidoc/src/05_building_block_view.adoc [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+=== Niveau 2
+
+[role="arc42help"]
+****
+Ici, vous pouvez spécifier la structure interne de (certaines) briques du niveau 1 sous forme de boîtes blanches.
+
+Vous devez décider quels éléments de votre système sont suffisamment importants pour justifier une description aussi détaillée.
+Préférez la pertinence à l'exhaustivité. Spécifiez les éléments importants, surprenants, risqués, complexes ou volatils.
+Laissez de côté les éléments normaux, simples, ennuyeux ou standardisés de votre système.
+****
```
[the remark this answers is not in the document]
Comment (dlucasd) [reply FR/asciidoc/src/05_building_block_view.adoc:146]: Sure, the third part is redundant. I change that.

**Comment by dlucasd** [comment]
Hi @gernotstarke , @martinweck ,

I've made the corrections in response to the @martinweck's code review.

**Comment by martinweck** [comment]
Hi @dlucasd,
Thanks a lot for those updates! For me, it is perfect now and can be merged.

**Comment by martinweck** [comment]
@gernotstarke github displays "1 workflow awaiting approval", maybe you can look into it and then you can merge it (I think I don't have the rights)

**Review by gernotstarke — APPROVED** [review]
thanx everybody

**Comment by gernotstarke** [comment]
@dlucasd I gave you triage-rights on the repo, so I can assign you as reviewer to Pull-Requests...

Reason: We received a few comments to your french translation - and my French is too bad to comment on those :-)

Would be great if you could accept the triage invite, so I can assign you to this new PR...

**Pull request merged**